src/main/java/run/halo/app/model/params/PostQuery.java

```diff
@@ -20,11 +22,22 @@
     /**
      * Post status.
      */
-    private PostStatus status;
+    private Set<PostStatus> status;
 
     /**
      * Category id.
      */
     private Integer categoryId;
 
+    public void setStatus(PostStatus... status) {
```

Comment:
IMO, we don't need this method here. If we wanted to pass statuses into `setStatuses` method, we could use [Set.of](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Set.html#of(E...)) utility.

Reply:
because the variable parameter method of the same name covers the setter method of status, I explicitly declare it.

Reply:
> IMO, we don't need this method here. If we wanted to pass statuses into `setStatuses` method, we could use [Set.of](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Set.html#of(E...)) utility.

because the variable parameter method of the same name covers the setter method of status, I explicitly declare it.